app/src/org/commcare/activities/CommCareVerificationActivity.java

```diff
@@ -276,6 +282,9 @@ public CustomProgressDialog generateProgressDialog(int taskId) {
                             Localization.get("verification.checking"),
                             taskId);
             dialog.addProgressBar();
+            if (fromSettings || fromManager) {

```

Comment:
Using the master build, I just tried to install an app via barcode w/ the 'include multimedia' unchecked. It didn't even show a 'verify multimedia' dialog. That makes me think he was referring to manually triggering multimedia. The reproduction steps from the ticket are also:

- Go to Settings
- Validate Media and the 'verifying multimedia' box automatically comes up.
- you used to be able to exit out of this by pushing the back button.